### PR TITLE
Bug 1430904 - Allow editing of a hook's triggerSchema, too

### PR DESCRIPTION
**Pull request by djmitche**
/cc @alexandrasp

### CODE REVIEW

File: src/views/HooksManager/HookDisplay.js [hunk 1]
```diff
@@ -76,10 +76,21 @@ export default class HookDisplay extends React.PureComponent {
           refreshHookStatus={refreshHookStatus}
         />
         <dl className="dl-horizontal">
-          <dt>Task Definition</dt>
-          <dd />
+          <dt>Task Template</dt>
+          <dd>
+            When the hook fires, this template is rendered with <a
+            href="https://taskcluster.github.io/json-e/">JSON-e</a> to create the
```
Comment (eliperelman) [inline src/views/HooksManager/HookDisplay.js]: Put external links in `target="_blank"` and add `rel="noopener noreferrer"`.

File: src/views/HooksManager/HookDisplay.js [hunk 2]
```diff
@@ -76,10 +76,21 @@ export default class HookDisplay extends React.PureComponent {
...
+            task definition. See <a
+            href="https://docs.taskcluster.net/reference/core/taskcluster-hooks/docs/firing-hooks">"firing
```
Comment (eliperelman) [inline src/views/HooksManager/HookDisplay.js]: Same for this link.

File: src/views/HooksManager/HookDisplay.js [hunk 3]
```diff
@@ -76,10 +76,21 @@ export default class HookDisplay extends React.PureComponent {
...
+            hooks"</a> for more information.
+          </dd>
         </dl>
         <Code language="json">{JSON.stringify(hook.task, null, 2)}</Code>
+        <dl className="dl-horizontal">
+          <dt>Trigger Schema</dt>
+          <dd>The payload to <tt>triggerHook</tt> must match this schema.</dd>
```
Comment (eliperelman) [inline src/views/HooksManager/HookDisplay.js]: Switch `tt` to `code`.

File: src/views/HooksManager/HookEditor.js [hunk 4]
```diff
@@ -33,6 +33,11 @@ const initialHook = {
       owner: 'name@example.com',
       source: 'https://tools.taskcluster.net/hooks'
     }
+  },
+  triggerSchema: {
```
Comment (eliperelman) [inline src/views/HooksManager/HookEditor.js]: Hmm, weird formatting here.

File: src/views/HooksManager/HookEditor.js [hunk 5]
```diff
@@ -72,7 +79,8 @@ export default class HookEditor extends React.PureComponent {
     ) {
       this.setState({
         hook: clone(this.props.isCreating ? initialHook : this.props.hook),
-        hookValidJson: true
+        hookValidJson: true,
+				triggerSchemaValidJson: true
```
Comment (eliperelman) [inline src/views/HooksManager/HookEditor.js]: Overindented.

File: src/views/HooksManager/HookEditor.js (hunk 4 above)
Comment (djmitche) [reply src/views/HooksManager/HookEditor.js]: How so?  It looks pretty normal to me..

Comment (eliperelman) [reply src/views/HooksManager/HookEditor.js]: Your patch superseded my review comments.

**Pull request merged**